docs/src/contributing_federation_development.md

```diff
@@ -47,11 +61,12 @@ Follow the normal installation instructions, either with [Ansible](administratio
 ```
```

Comment:
Btw the federation image hasnt been updated in two months. It should work fine with the normal release images, as they contain all the necessary code.